snap/snapcraft.yaml

```diff
@@ -256,14 +256,8 @@ parts:
       - -*
 
   qemu:
-    override-pull: |
-      versions_url=https://raw.githubusercontent.com/kata-containers/runtime/${SNAPCRAFT_PROJECT_VERSION}/versions.yaml
-      branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
-      url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.url)"
-      curl -LO ${url}/archive/${branch}.tar.gz
-      tar -xf ${branch}.tar.gz --strip-components=1
     plugin: make
-    after: [qemu-scripts-dump, qemu-patches-dump]
+    after: [go, qemu-scripts-dump, qemu-lite-patches-dump, qemu-aarch64-patches-dump]
```

Comment:
go is not required to build qemu

Reply:
Agree, golang should not be required for building qemu.